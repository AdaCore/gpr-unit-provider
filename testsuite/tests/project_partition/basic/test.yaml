description:
    A test for basic project partition
driver: build_and_run
project_file: default.gpr
main: main
