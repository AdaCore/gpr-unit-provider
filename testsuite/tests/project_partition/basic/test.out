Loading the project:
[LIBADALANG.GPR2_PROVIDER] Trying to partition AP1
   [LIBADALANG.GPR2_PROVIDER] Found conflicting source files for unit P1 in P3 and <P1, P2>
[LIBADALANG.GPR2_PROVIDER] Input project partitionned into:
   [LIBADALANG.GPR2_PROVIDER] Part 1: <P1, P2>
   [LIBADALANG.GPR2_PROVIDER] Part 2: <P3>
  * P1 P2
  * P3

<DottedName p2.ads:4:24-4:28> resolves to <ObjectDecl ["I"] p1.ads:2:4-2:16>
Done.
